### ISSUE TITLE
PHP8: Attributes support

### ISSUE THREAD

**Issue by Wulfheart** [glayzzle/php-parser#654]
#486

**Comment by MasonD** [comment]
Attributes are kinda supported on the php8 branch, but not completely. For example, the following throws an error:
```php
#[Attribute(Attribute::TARGET_PROPERTY | Attribute::TARGET_METHOD)]
class ClassName { }
```
Error: Bad terminal sequence "|"

**Comment by czosel** [comment]
Fixed in `3.1.0-beta.6` :tada:
